point_pattern.py

```diff
@@ -56,9 +57,15 @@ def find_largest_city(gj):
     population : int
                  The population of the largest city
     """
-    city = None
+    temp = gj['features']
+    city = ""
     max_population = 0
 
+    for i in temp:
+        if (i['properties']['pop_max'] > max_population):
+            max_population = i['properties']['pop_max']
+            city = i['properties']['name']
+
     return city, max_population
```

Comment:
Kind of, so far I am pulling all the tweets from the the Twitter API and parsing the JSON for a handful of attributes and then writing them to a .csv (one column per tweet). However, when I start pulling from the Twitter Stream, it might be beneficial to just save summary stats from the JSON instead of every individual tweet, since I am not interested in everything from the stream data.